src/app/static/src/js/components/workflowWizard/workflowWizard.vue

```diff
@@ -16,6 +16,11 @@
                        @update="updateMetadata">
             </component>
         </step>
+        <div id="run-workflow-status" v-if="runningStatus"
```

Comment:
I'm not convinced this belongs in the wizard - feels like it's the responsibility of the finalise step to show this -  that would mean an extra level of emitting, but you would avoid having to manage `runningStatus` here.

Reply:
I am not sure I know how to display the progress message on runWorkflowRun step before the buttons are rendered or perhaps not possible. The wizard displays runWorkflowRun page and add navigation buttons underneath, so adding a progress on this page will mean the text will display before navigation buttons.

Reply:
That's true, I guess if it's under the buttons then it's in the domain of the wizard...